ktor-http/ktor-http-cio/common/test/io/ktor/tests/http/cio/HttpParserTestUtils.kt

```diff
@@ -0,0 +1,27 @@
+/*
+* Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+*/
+
+package io.ktor.tests.http.cio
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.intrinsics.*
+import kotlin.coroutines.*
+import kotlin.test.*
+
+@OptIn(InternalCoroutinesApi::class)
+internal fun test(block: suspend () -> Unit) {
```

Comment:
why not just use `testSuspend`?

Reply:
IMO, it will be better to use `testSuspend`, to be more consistent. I think, that there is no reason to keep different suspend test runner functions all over the codebase.